src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -443,16 +443,54 @@ pass::EliminateConvert::EliminateConvert() {
         if (!convert) {
             return false;
         }
+
         if (convert->get_input_element_type(0) == convert->get_element_type()) {
             return replace_output_update_name(convert->output(0), convert->input_value(0));
         }
+
+        // Only for some test.
```

Comment:
This is definetelly incorrect way to solve test failures. Test exptected behavior should be updated instead.